src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { RouteComponentProps } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Remove this and add the 2fa code logic
+const doubleAuthCode = ''
+
+export const LegacyLogin = ({ history }: RouteComponentProps) => {
+  const captchaBgColor = useColorModeValue('gray.50', 'gray.700')
+  const checkboxBorderColor = useColorModeValue('gray.400', 'white')
+
+  const onSubmit = async (values: FieldValues) => {
+    return
+  }
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors, isSubmitting },
+  } = useForm({ shouldUnregister: true })
+
+  const translate = useTranslate()
+
+  return (
+    <>
+      <ModalHeader>
+        <Text translation={'walletProvider.shapeShift.legacy.loginWithAccount'} />
+      </ModalHeader>
+      <ModalBody pt={0}>
+        <Text
+          color='gray.500'
+          mb={4}
+          translation={'walletProvider.shapeShift.legacy.loginInformations'}
+        />
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <FormControl isInvalid={errors.email} mb={4}>
+            <Input
+              {...register('email', {
+                pattern: {
+                  value: /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/i,
+                  message: translate('walletProvider.shapeShift.legacy.invalidEmail'),
+                },
+                required: translate('walletProvider.shapeShift.legacy.emailRequired'),
+              })}
+              type='text'
+              placeholder={translate('walletProvider.shapeShift.legacy.emailAddress')}
+              variant='filled'
+              height={12}
+            />
+            <FormErrorMessage>{errors.email?.message}</FormErrorMessage>
+          </FormControl>
+          <FormControl isInvalid={errors.password}>
+            <Input
+              {...register('password', {
+                required: translate('modals.shapeShift.password.error.required'),
+              })}
+              type='password'
+              placeholder={translate('walletProvider.shapeShift.legacy.password')}
+              variant='filled'
+              autoComplete='off'
+              height={12}
+            />
```

Comment:
Will this be removable from form state once we decrypt the response?

Reply:
I provided an example on how to reset form fields :+1: